apollo-router/src/query_planner/tests.rs

```diff
@@ -1458,9 +1455,7 @@ async fn typename_propagation2() {
                         "representations": [{
                             "__typename": "Book",
                             "bookId": "book1",
-                            "author": {
-                                "fullName": "Ada"
-                            }
+                            "author": null
```

Comment:
why this change?

Reply:
Because apparently that's the query planner is asking for. If I don't do this I have an error telling me that no mock request is matching

Reply:
right, I saw that, it's ok